src/liballoc/str.rs

```diff
@@ -431,6 +431,13 @@ impl str {
     ///
     /// assert_eq!(new_year, new_year.to_uppercase());
     /// ```
+    ///
+    /// One character can become multiple:
+    /// ```
+    /// let s = "tschüß";
```

Comment:
Fun fact: uper-case `ß` [is a thing nowadays](https://en.wikipedia.org/wiki/Capital_%E1%BA%9E). Not sure if Unicode will ever want to use that for `'ß'.to_uppwercase()` though. ;)